ONSdigital/takeon-ui
# Feature/spp 1515 se override

# Description
- Added function to send override notification to API Gateway
- Added function to put the data in the correct format that the API Gateway/Queue expects
- Added validate_json function to helpers.py
- Added unit tests for new functions

# How to test?
- Test by pressing the override button in spp1515 UI, check cloudwatch logs to see any processing and check if the output from the UI reaches the **spp-es-takeon-validation-spp1515-sendbpm** queue.

# Links
Jira Link: https://collaborate2.ons.gov.uk/jira/browse/SPP-1515
